lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -1085,6 +1085,7 @@ protected function initializeDoctrineTypeMappings()
             'binary'        => 'binary',
             'varbinary'     => 'binary',
             'set'           => 'simple_array',
+            'bit'           => 'integer',
```

Comment:
Looks like entries are grouped by values. Could you move this line to line 1062?

Reply:
Ok, I have already done that.

Reply:
> Please squash your 2 commits together, we don't need to know it was not on the right line from the start, and it's hard to understand what you are fixing with a commit message that just says "fix"

Ok, please check the latest commit